deploy/helm/charts/README.md

```diff
@@ -57,11 +56,9 @@ The following table lists the configurable parameters of the OpenEBS NDM chart a
 | `ndmOperator.nodeSelector`              | Nodeselector for operator pods                | `""`                                      |
 | `ndmOperator.tolerations`               | NDM operator's pod toleration values          | `""`                                      |
 | `ndmOperator.securityContext`           | Seurity context for container                 | `""`                                      |
-| `ndmOperator.healthCheck.initialDelaySeconds`   | Delay before liveness probe is initiated      | `30`                              |
-| `ndmOperator.healthCheck.periodSeconds`         | How often to perform the liveness probe       | `60`                              |
-| `ndmOperator.readinessCheck.initialDelaySeconds`| Delay before readiness probe is initiated     | `4`                               |
-| `ndmOperator.readinessCheck.periodSeconds`      | How often to perform the readiness probe      | `10`                              |
-| `ndmOperator.readinessCheck.failureThreshold`   | Failure threshold for the readiness probe     | `1`                               |
+| `featureGates.APIService.enabled`       | Enable the gRPC API service of NDM            | `false`                                   |
+| `featureGates.UseOSDisk.enabled`        | Enable feature-gate to use free space on OS disk | `false`                                   |
+| `featureGates.MountChangeDetection.enabled` | Enable feature-gate to detect mountpoint/filesystem changes | `false`                                   |
```

Comment:
What are the pending tasks to get this enabled by default?

Reply:
Automatic capacity change detection should also be added. Also thought that we will keep it default disabled for 2-3 releases and then move to default enabled, so that people who want to try can test this feature.